qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -12,6 +12,9 @@
 """
 from __future__ import (absolute_import, division, print_function, unicode_literals)
 
+# Importing resources loads the data in
```

Comment:
Could you add a comment as to indicate why this has to be done so early? I can imagine someone will try to tidy up and move this.